__integrations__/models/service_preference.test.ts

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable no-console */
+import { FiscalCode, NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+
+import * as TE from "fp-ts/lib/TaskEither";
+import * as E from "fp-ts/lib/Either";
+import {
+  ServicePreference,
+  SERVICE_PREFERENCES_MODEL_PK_FIELD,
+  SERVICE_PREFERENCES_COLLECTION_NAME,
+  ServicesPreferencesModel,
+  NewServicePreference,
+  AccessReadMessageStatusEnum
+} from "../../src/models/service_preference";
+import { createContext } from "./cosmos_utils";
+import { pipe } from "fp-ts/lib/function";
+import { DocumentSearchKey } from "../../src/utils/cosmosdb_model";
+import { NonNegativeInteger } from "@pagopa/ts-commons/lib/numbers";
+import { withoutUndefinedValues } from "@pagopa/ts-commons/lib/types";
+
+const aFiscalCode = "FRLFRC74E04B157I" as FiscalCode;
+const aServiceId = "aServiceId" as NonEmptyString;
+
+const aServicePreference: ServicePreference = pipe(
+  ServicePreference.decode({
+    fiscalCode: aFiscalCode,
+    serviceId: aServiceId,
+    accessReadMessageStatus: AccessReadMessageStatusEnum.ALLOW,
+    isEmailEnabled: true,
+    isInboxEnabled: true,
+    settingsVersion: 0 as NonNegativeInteger,
+    isWebhookEnabled: true
+  }),
+  E.getOrElseW(() => {
+    throw new Error("Cannot decode service payload.");
+  })
+);
```

Comment:
I think that is overkill this tipe of definition. IMHO make sense if we use the with default capabilities like for the `Profile`.
```suggestion
const aServicePreference: ServicePreference = {
  fiscalCode: aFiscalCode,
  serviceId: aServiceId,
  accessReadMessageStatus: AccessReadMessageStatusEnum.ALLOW,
  isEmailEnabled: true,
  isInboxEnabled: true,
  settingsVersion: 0 as NonNegativeInteger,
  isWebhookEnabled: true
};
```

Reply:
simplified in https://github.com/pagopa/io-functions-commons/pull/270/commits/2c5fe322763ae40ed11b6e23fe10a60620fb2835